### PR TITLE
feat: add new link fields + fix contribution type

### PR DESCRIPTION
**Pull request by marcuspang**
To accommodate https://github.com/RadarDAO-xyz/radar-launch-frontend/issues/239

Note that `video` isn't currently being used

### CODE REVIEW

File: src/models/Pool.ts
Comment (mrTomatolegit) [inline src/models/Pool.ts:1]: If you're updating the schema you need to update the ts types as well. Lines 3 and 10

**Review by mrTomatolegit — CHANGES_REQUESTED** [review]
Outdated typescript types, not equivalent to schema descriptor

**Comment by marcuspang** [comment]
@mrTomatolegit added a new hidden field to determine whether it'll be shown when choosing which brief to submit a project to

**Review by mrTomatolegit — APPROVED** [review]
lgtm

**Pull request merged**